### ISSUE TITLE
Alternative - New Keyboard Shortcuts for tabs  "reopen" and "reverse navigation" And Zoom In Zoom Out 

### ISSUE THREAD

**Issue by just-one-android** [brave/brave-ios#7592]
### Description: 
This bug (or lack of functionality) has been reported in the community forum, but was autoclosed, although the report's author appears to have attempted to keep it open.
https://community.brave.com/t/tab-shortcuts-do-not-match-safari/472522

I experience the same issue using the same hardware. The `ctrl shift tab` does not navigate tabs in reverse, and `ctrl shift t` does not reopen a tab after closing.

The functionality does exist in both Safari and Google Chrome on iPadOS.

Edit: Additionally 

Zoom In `Cmd +`
Zoom Out `Cmd -`

### Brave Version:
Brave Version 1.51.1 (23.5.19.19)

### Device details:
iPadOS 16.5 (20F66)
iPad Pro (12.9-inch) (5th generation)

**Comment by soner-yuksel** [comment]
Both of the actions exist in Brave 

Navigate tabs in reverse is using `.cmd - option - left`

Reopen a tab after closing is `cmd - shift  - t`

However we have no conflicts for 

`ctrl shift tab `-> navigate tabs in reverse, and `ctrl shift t` for a tab after closing so they will be added as alternative to main ones which will work together.

**Comment by just-one-android** [comment]
Hello, thanks for the update. The same problem still persists for me in Brave on the new app version and iPadOS version reported below.

The `ctrl shift tab` shortcut does not navigate tabs in reverse, and the `cmd option left` will reverse, but cannot cycle. The behavior is not the same as with Safari or Google Chrome.

The `cmd shift t` shortcut does not reopen a tab after closing _using the `cmd w` shortcut_, but does work after using the tab X button with the touchcreen. This is also a different behavior than other browsers.


### Brave Version:
Brave Version 1.58.2 (23.11.3.10)

### Device details:
iPadOS 17.1.1 (21B92)
iPad Pro (12.9-inch) (5th generation)

**Comment by soner-yuksel** [comment]
@just-one-android please check out after the changes. The shortcuts you wanted will be operating when that PR is merged and tit is added to 1.61

`ctrl shift tab -> navigate tabs in reverse, and ctrl shift t for a tab after closing `

**Comment by hffvld** [comment]
Verified on `iPad Air` using version(s):
```
Device/OS: iPad Air / iPadOS 16.7.2
Brave build: 1.61 (23.12.11.17)
BraveCore: 1.61.102 (120.0.6099.71)
```

Filed follow-up issue #8552
---------------------------

**STEPS:**
1. Launch Brave 
2. Open a few tabs
3. Use `Ctrl` + `Tab` > Verify that forward tab navigation is working
4. Use `Ctrl` + `Shift` + `Tab` > Verify that reverse tab navigation is working
5. Close the tab using `X` on the tab in Brave or using shortcut `Cmd` + `W`
6. Use `Ctrl` + `Shift` + `T` > Verify that closed tab is restored
7. Open `Wikipedia` in the new tab
8. Use `Cmd` + `+` to zoom in and `Cmd` + `-` to zoom out > Verify that `zoom in` and `zoom out` is working

**ACTUAL RESULTS:**
- Verified that forward tab navigation is working when using `Ctrl` + `Tab` or `Cmd` + `Option` + `→`
- Verified that reverse tab navigation is working when using `Ctrl` + `Shift` + `Tab` or `Cmd` + `Option` + `←`
- Verified that the closed tab can be restored when using `Ctrl` + `Shift` + `T` or `Cmd` + `Shift` + `T`
- Confirmed that in Safari and Chrome, closed tabs can be restored using **ONLY** `Cmd` + `Shift` + `T`, but not `Ctrl` + `Shift` + `T` when tabs are closed with shortcut `Cmd` + `W`. This is still not working in Brave.
- Verified that `zoom in` and `zoom out` is working when using `Cmd` + `+` to zoom in and `Cmd` + `-` to zoom out

---------------------------

*Brave. Tab navigation using `Cmd` + `Option` + `←` or `→` and `Ctrl` + `Tab` or `Ctrl` + `Shift` + `Tab`*

https://github.com/brave/brave-ios/assets/128532965/b458087d-1be3-4d57-851c-a54e422ec522

*Brave. Reopen the closed tab using `Cmd` + `Shift` + `T` or `Ctrl` + `Shift` + `T`*

https://github.com/brave/brave-ios/assets/128532965/6c06117f-fd83-494b-988e-b1f63d9e0181

*Safari and Chrome. Reopen the closed tab using `Cmd` + `Shift` + `T` or `Ctrl` + `Shift` + `T`*

https://github.com/brave/brave-ios/assets/128532965/808a87d3-8728-4250-97db-8cbe4d5ef1b4

*Brave. `Zoom in` and `zoom out` using `Cmd` + `+` and `Cmd` + `-`*
